common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -54,6 +54,14 @@ def create_modulestore_instance(
     )
 
 
+def mock_tab_from_json(tab_dict):
```

Comment:
It's tested more thoroughly in the `lms` tests, specifically `courseware/tests/test_tabs.py`. The reason we have to mock it out here is because of the way the plugin discovery doesn't work within the `xmodule` tests. Most of these tests only check to see that the right number of tabs have been returned.